aerospike/content.md

```diff
@@ -173,11 +173,11 @@ These images are based on [debian:strech-slim](https://hub.docker.com/_/debian).
 
 ### ee-[version]
 
-These tags are for Aerospike EE images, and will require a feature key file, such as the one you get with the single-node EE evaluation, or one associated with a commercial enterprise license agreement.
+These [tags](https://hub.docker.com/r/aerospike/aerospike-server-enterprise/tags?page=1&ordering=last_updated) are for Aerospike EE images, and will require a feature key file, such as the one you get with the single-node EE evaluation, or one associated with a commercial enterprise license agreement.
 
 ### ce-[version]
 
-These tags are for Aerospike CE images, and do not require a feature key file to start. As mentioned above, the developer API for both is the same, but the editions differ in operational features.
+These [tags](https://hub.docker.com/r/aerospike/aerospike-server/tags?page=1&ordering=last_updated) are for Aerospike CE images, and do not require a feature key file to start. As mentioned above, the developer API for both is the same, but the editions differ in operational features.
```

Comment:
```suggestion
These [tags](https://hub.docker.com/_/aerospike?tab=tags&ordering=last_updated&name=ce-) are for Aerospike CE images, and do not require a feature key file to start. As mentioned above, the developer API for both is the same, but the editions differ in operational features.
```

Reply:
@tianon the official repo seems to have reverted to the old docker store publisher. Any changes occurring. We use to update content from updating content.md Have thing changed?

Reply:
My guess is that it's probably related to https://github.com/docker-library/docs/issues/1899 in some way -- I'll ask again. :disappointed: